### ISSUE TITLE
API versioning?

### ISSUE THREAD

**Issue by paralaxsd** [pouetnet/pouet-api#2]
Hi!
We're looking into integrating the Pouet API with our demo launcher UI.
This is an easy enough task but there's one thing that we have to be able to rely on: a stable API.

Unfortunately http://api.pouet.net/ states: "_More to come, mostly on demand. Data structures are subject to change without notice._"

since most demos we release will end up hosted on scene.org we're not able to update our client code once the demo's out in the wild.

Fortunately there's one easy thing to solve this problem for all of us: use versioning.
The current pouet API is available on http://api.pouet.net/v1. Introducing a new version of the API with breaking changes under http://api.pouet.net/v2 would be a sensible step to keep old clients in the game. 

Is this something you would consider?

Thanks!

**Comment by Gargaj** [comment]
That's the hope at least, yes. The API is currently being expanded, but it's unlikely that fields get moved or removed.

**Comment by Gargaj** [comment]
Also you can always update client code too - see https://github.com/Gargaj/Conduit

**Comment by paralaxsd** [comment]
Yeah, adding new fields and endpoints won't break v1 so that would be totally fine.
